idp_data/idp_data/urls.py

```diff
@@ -1,8 +1,7 @@
 from django.urls import path
-
 from . import views
 
 
 urlpatterns = [
-    path("", views.Index.as_view(), name="index"),
```

Comment:
actually we want to keep trailing commas and start running `docker-compose run --rm web black .` to normalise the code

Reply:
done